compiler/bsc/rescript_compiler_main.ml

```diff
@@ -402,6 +402,10 @@ let command_line_flags : (string * Bsc_args.spec * string) array =
     ( "-absname",
       set absname,
       "*internal* Show absolute filenames in error messages" );
+    ( "-enable-experimental",
```

Comment:
I can imagine this might grow in the future.
Should this be a list to be future proof?

Reply:
It is a list, just that you pass it multiple times. Or are you referring to making it `-enable-experimental FeatureX,FeatureY` instead of `-enable-experimental FeatureX -enable-experimental FeatureY`?